decoder.c

```diff
@@ -1068,6 +1068,12 @@ SRD_API int srd_decoder_load_all(void)
 	return SRD_OK;
 }
 
+static void srd_decoder_unload_cb(void *arg, void *ignored)
+{
+	ignored = ignored; // Prevent unused warning
+	srd_decoder_unload((struct srd_decoder *)arg);
```

Comment:
Please use (void)ignored instead

Reply:
@abraxa would you prefer an additional commit for the change, or amend & force-push?

Reply:
I'd prefer amending so that we have a "clean" commit. Thank you!